gitlab/pkg/reconciler/gitlabsource.go

```diff
@@ -409,7 +305,7 @@ func (r *Reconciler) getOwnedKnativeService(source *sourcesv1alpha1.GitLabSource
 }
 
 // UpdateFromLoggingConfigMap loads logger configuration from configmap
-func (r *Reconciler) UpdateFromLoggingConfigMap(cfg *corev1.ConfigMap) {
+func (r *Reconciler) UpdateFromLoggingConfigMap(cfg *corev1.ConfigMap) { // TODO: remove this.
```

Comment:
This is core functionality in other places and should be handled at the controller level, not the reconciler. See eventing and serving on how to do this.